scripts/package-artifacts.ps1

```diff
@@ -29,6 +29,8 @@ function Package-NetScanner([string]$sourcetfm, [string]$targettfm)
     Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\$sourcetfm\SonarScanner.MSBuild.PreProcessor.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
     Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\$sourcetfm\SonarScanner.MSBuild.runtimeconfig.json" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
     Copy-Item -Path "$PSScriptRoot\..\src\SonarScanner.MSBuild\bin\Release\$sourcetfm\SonarScanner.MSBuild.Shim.dll" -Destination "$fullBuildOutputDir\sonarscanner-msbuild-$targettfm"
+    # Take files from netcoreapp3.1 because they are not present in netcoreapp2.1 bin directory
```

Comment:
No, I don't know why the hell it's not present there 🤷 
I'm mechanically following what was done for `Newtonsoft.Json.dll` on the line below.

Reply:
Can't we take them all from `netstandard`?

Reply:
That's not an option, They are coming from `bin` folder. And there we have `net46`, `net5.0`, `netcoreapp2.1` and `netcoreapp3.1` according to our target frameworks.
Once we change targeting to target only `netstandard2.0` in MMF-2918 (reduce S4NET versions), this problem will disappear.